UnitTests/MockAPIClient.swift

```diff
@@ -0,0 +1,3 @@
+@objc class MockAPIClient: BTAPIClient {
```

Comment:
Right now this doesn't really do anything, but I think in the future it will be helpful to have a `MockAPIClient` in Drop-in.